miio/integrations/petfeeder/status.py

```diff
@@ -0,0 +1,85 @@
+from datetime import timedelta
+from enum import Enum
+from typing import Any, Dict
+
+from miio import DeviceStatus
+
+
+class FoodLevel(Enum):
+    Normal = 0
+    Low = 1
+    Empty = 2
+
+
+class FoodOutletStatus(Enum):
```

Comment:
Simply use a bool for boolean values, it's not worth to make it more complicated.